### ISSUE TITLE
missed "onwarn"?

### ISSUE THREAD

**Issue by dadiborn** [EMH333/esbuild-svelte#85]
I want to use this tip from the link, but looks like there is no support for **onwarn** in 'esbuild-svelte'?
https://github.com/sveltejs/svelte/issues/5300#issuecomment-864687861

**Comment by EMH333** [comment]
Correct, that appears to be a feature implemented by the Svelte Rollup plugin, not by Svelte itself so it isn't included here

**Comment by EMH333** [comment]
Released as `v0.6.3`
